generate.go

```diff
@@ -30,11 +30,11 @@ func (c Cfg) GenerateValue() string {
 	if c.encrypted {
 		// TODO COGS-1657
 		// decrypt.File(c.Path, c.SubPath)
-		return "|enc| " + c.Path
+		return "|enc|" + c.Path
 	}
 	// TODO COGS-1659
 	// cogs.File(c.Path, c.SubPath)
-	return "path." + c.Path + "/" + c.SubPath
+	return "|path|" + c.Path + "|subpath|" + c.SubPath
```

Comment:
just curious, should we check that subpath is present before adding? or is it ok to possible have an empty subpath here?

Reply:
It possible to have an empty subpath, my intent here is that a large majority of our configs will be themselves flat k/v pairs. Subpath should not point to the value of the key but rather the object that holds the k/v mapping (or a representation of it) for now, it is just a placeholder idea.